### ISSUE TITLE
Output time data to file

### ISSUE THREAD

**Issue by matthewrsj** [osuosl/climesync#23]
**Feature Idea**
Allow user to output time data to a file if they so choose. This should probably be output as JSON so it's standardized. A simple call to `json.dump` should do it.

**Comment by Kennric** [comment]
Interesting idea, and possibly in parallel with a clock-in/clock-out feature which would save entries up all day then submit in bulk.

**Comment by aaroncohen73** [comment]
Superseded by #70
